Core/ResponseHandler.php

```diff
@@ -122,7 +122,7 @@ private static function _isPostProcessingAction($oResponse)
      *
      * @since 1.0.0
      */
-    private static function _saveTransaction($oResponse, $oOrder, $oBackendService)
+    public static function saveTransaction($oResponse, $oOrder, $oBackendService)
```

Comment:
Why do you need this public now?

Reply:
We had to use the function in NotifyHandler.php.